holidays/countries/senegal.py

```
@@ -0,0 +1,157 @@
+
+        # Assumption Day.
+        self._add_assumption_of_mary_day(tr("Assomption"))
+
+        # All Saints' Day.
+        self._add_all_saints_day(tr("Toussaint"))
+
+        # Christmas Day.
+        self._add_christmas_day(tr("Noël"))
+
+
+class SN(Senegal):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)